### PR TITLE
OCLOMRS-143: Redesign page to match original mockup

### PR DESCRIPTION
**Pull request by emasys**
# JIRA TICKET NAME:
[Redesign page to match original mockup](https://issues.openmrs.org/browse/OCLOMRS-143)

# Summary:
Redesign the add concept page to match the provided mockup.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -20,6 +20,7 @@ import GeneralSearchContainer from './components/GeneralSearch/GeneralSearchCont
 import UserDashboard from './components/userDasboard/container/UserDashboard';
 import LoginDetails from './components/login/container/LoginDetails';
 import AddBulkConcepts from './components/bulkConcepts/addBulkConcepts';
+import BulkConceptPage from './components/bulkConcepts/container/BulkConceptsPage';
```
Comment (hadijahkyampeire) [inline src/App.js]: I think this should be `CielConceptPage` as opposed to `BulkConceptPage` since that's a different page.

Comment (emasys) [reply src/App.js]: The page will eventually render Bahmni concepts as well, so I think the name is correct. You may refer to the mockup for more clarity.

Comment (hadijahkyampeire) [reply src/App.js]: So will the heading of the page

**Comment by emasys** [comment]
The ticket name is already the same as the PR name. @shakirandagire

**Comment by shakirandagire** [comment]
@emasys "OCLOMRS-143: Redesign add bulk concept page" is not the same as " OCLOMRS-143
Redesign page to match original mockup"

**Comment by alexmochu** [comment]
@emasys Your PR name does not match your Jira Ticket Name

**Pull request merged**